src/main/java/frc/team2412/robot/subsystem/PostClimbSubsystem.java

```diff
@@ -0,0 +1,28 @@
+package frc.team2412.robot.subsystem;
+
+import static frc.team2412.robot.Hardware.*;
+
+import edu.wpi.first.wpilibj.PneumaticsModuleType;
+import edu.wpi.first.wpilibj.Solenoid;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import io.github.oblarg.oblog.Loggable;
+
+public class PostClimbSubsystem extends SubsystemBase implements Loggable {
+
+    private Solenoid upwardsSolenoid;
+    private Solenoid downwardsSolenoid;
+
+    public PostClimbSubsystem() {
+        upwardsSolenoid = new Solenoid(PNEUMATIC_HUB, PneumaticsModuleType.REVPH, POST_CLIMB_SOLENOID_UPWARDS);
+        downwardsSolenoid = new Solenoid(PNEUMATIC_HUB, PneumaticsModuleType.REVPH, POST_CLIMB_SOLENOID_DOWNWARDS);
+    }
+
+    public void goingUp() {
+        upwardsSolenoid.set(true);
+    }
+
+    public void goingDown() {
+        downwardsSolenoid.set(true);
+    }
```

Comment:
The names aren't terribly clear as to what these actually do.  I'd prefer it to be easy to translate the method names to physical actions on the robot. Since we're not a jumping bot, I'm pretty sure that activating a solenoid doesn't directly make the robot go up. Last I saw, these actually activate/engage (or deactivate/release/disengage) a lock on a gear.